app/src/main/java/com/example/home/User.java

```diff
@@ -0,0 +1,28 @@
+package com.example.home;
+
+
+import org.json.JSONObject;
+import org.parceler.Parcel;
+
+import java.util.ArrayList;
+@Parcel
+public class User {
+    public String userName;
+    public String password;
+    public String email;
+    public ArrayList<Homes> savedHomes;
+    public String location;
```

Comment:
Instead of having everything in the User Object. I'd recommend creating another object called "UserPreferences", and you can keep noofBedrooms,noOfBathrooms, houseStyle in that object and user can simply have a reference to this userpreferences object